music_assistant/providers/audiobookshelf/__init__.py

```diff
@@ -422,11 +521,21 @@ async def _get_stream_details_podcast_episode(self, podcast_id: str) -> StreamDe
     async def on_played(
         self, media_type: MediaType, item_id: str, fully_played: bool, position: int
     ) -> None:
-        """Update progress in Audiobookshelf."""
+        """Update progress in Audiobookshelf.
+
+        In our case media_type may have 3 values:
+            - PODCAST
+            - PODCAST_EPISODE
+            - AUDIOBOOK
+        We ignore PODCAST (function is called on adding a podcast with position=None)
```

Comment:
Yes, the backend function just forwards all the internal playlog entries but it indeed doesnt make sense to act on the "parent type" - just only act on audiobook and podcast_episode indeed.